recipes/lzstring/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+{% set name = "lzstring" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "d54dd5a5f86837ccfc1343cc9f1cb0674d2d6ebd4b49f6408c35104f0a996cb4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
+
+  run:
+    - python
+    - future
+
+test:
+  imports:
+    - lzstring
+
+about:
+  home: https://github.com/gkovacs/lz-string-python
+  summary: 'lz-string for python'
+  license: MIT
+  license_family: MIT
```

Comment:
Nice! Thanks for doing that. If we add the `MANIFEST.in`, we should be able to package the license here in the future as well.

Sorry for not being more specific about this, but could we add a comment to the `meta.yaml` about the license file. Here's a [typical comment]( https://github.com/conda-forge/pprofile-feedstock/blob/6d487b8ccaed562299116ab4730ef1e28d750e38/recipe/meta.yaml#L37-L41 ) of this sort. Though for the recipe in this PR the license file has a different name ( `LICENSE.md` ).